lib/liquid/block_body.rb

```diff
@@ -154,14 +154,22 @@ def render_to_output_buffer(context, output)
     private
 
     def render_node(context, output, node)
-      node.render_to_output_buffer(context, output)
+      return if node.disabled?(context, output)
```

Comment:
Much better! Just one last thing: I find it weird/unexpected that `node.disabled?` ends up mutating the `output`. How about:

```ruby
if node.disabled?(context)
  output << node.disabled_error_message
  return
end
```

Reply:
Yeah this more just comes from the original way of implementing in the original tag. I'll change it over